From 19164625f71bd8b28fbbfca46f6c8eb7c6ba22f4 Mon Sep 17 00:00:00 2001
From: Yann Gautier <yann.gautier@foss.st.com>
Date: Thu, 21 Dec 2023 09:51:40 +0100
Subject: [PATCH 475/954] refactor(st-clock): remove unused struct

The struct clk_fixed_rate is used nowhere in the code, remove its
definition.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/arm-trusted-firmware.git 1aafcd73d0767f4780108be25525cecfdd34eb7f refactor(st-clock): remove unused struct]
Change-Id: I139ad05a249357da96a996feabd4b1f53e290f2a
Signed-off-by: Yann Gautier <yann.gautier@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/tf-a/+/352181
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 drivers/st/clk/clk-stm32-core.h | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/drivers/st/clk/clk-stm32-core.h b/drivers/st/clk/clk-stm32-core.h
index 8f65f6b40e..ccf5a0728e 100644
--- a/drivers/st/clk/clk-stm32-core.h
+++ b/drivers/st/clk/clk-stm32-core.h
@@ -108,11 +108,6 @@ struct clk_oscillator_data {
 
 };
 
-struct clk_fixed_rate {
-	const char *name;
-	unsigned long fixed_rate;
-};
-
 struct clk_gate_cfg {
 	uint32_t offset;
 	uint8_t bit_idx;
-- 
2.53.0

